source/expressions.tex

```diff
@@ -5285,6 +5285,11 @@
 \grammarterm{expression} of a \grammarterm{noptr-new-declarator}), but
 \tcode{new float[5][n]} is ill-formed (because \tcode{n} is not a
 constant expression).
+Furthermore,
+\tcode{new float[0]} is well-formed
+(because \tcode{0} is an \grammarterm{expression}), but
```

Comment:
please use the same phrasing as the other sentence: "because 0 is the expression of a noptr-new-declarator"